src/Illuminate/Support/Facades/Route.php

```diff
@@ -3,6 +3,7 @@
 namespace Illuminate\Support\Facades;
 
 /**
+ * @method static \Illuminate\Routing\Route fallback(\Closure|array|string|callable|null $action = null)
```

Comment:
Why does the doc for a this (and the others) include callable and Closure. Should just be callable tbh?

Reply:
// cc @driesvints

Reply:
Should just be `Closure` since it's specifically checked in the `actionReferencesController` method on the Router object.